hack/release.sh

```diff
@@ -32,7 +32,9 @@ function build_release() {
   GOOS=darwin GOARCH=amd64 go build -mod=vendor -ldflags "${ld_flags}" -o ./kn-darwin-amd64 ./cmd/...
   echo "🚧 🎠 Building for Windows"
   GOOS=windows GOARCH=amd64 go build -mod=vendor -ldflags "${ld_flags}" -o ./kn-windows-amd64.exe ./cmd/...
-  ARTIFACTS_TO_PUBLISH="kn-darwin-amd64 kn-linux-amd64 kn-windows-amd64.exe"
+  echo "🚧 🐳 Building the container image"
+  ko publish ${KO_FLAGS} github.com/knative/client/cmd/kn 2>&1 | grep -E '(Loaded|Published)' | rev | cut -d ' ' -f 1 | rev > kn-image-location
```

Comment:
This will hide any error messages from `ko publish`. At a minimum, `tee` should be used, although I think this is a unnecessary convoluted and brittle way of extracting the location of the generated image: this can be easily derived from KO_REPO_PATH and the package name.

Reply:
It is brittle; https://github.com/google/ko/pull/36 is now merged and solves this problem, but `ko` isn't released with that yet.

Reply:
Could you elaborate more how to integrate ko publish with tee command?

Reply:
Just pipe it. Although I do not recommend, specially now that the problem is solved.

Reply:
@sixolet has a PR for ko https://github.com/google/ko/pull/36 already merged. @adrcunha I do not think we  need tee any more, since we can print in stdout.

Reply:
I still insist on not using such brittle approach. Instead I would recommend cleanly getting the hash through gcloud, like

```
gcloud container images describe gcr.io/XXXXXX/github.com/knative/client/cmd/kn:latest --format="value(image_summary.digest)"
```